src/firefly/js/core/JsonUtils.js

```diff
@@ -84,12 +84,11 @@ export const jsonRequest= function(baseUrl, cmd, paramList, doPost) {
                 return;
             }
             response.json().then( (result) => {
-                if (result && result[0]) {
-                    if (result[0] && result[0].success && result[0].success !== 'false' && result[0].data) {
-                        resolve(result[0].data);
-
+                if (has(result,'0')) {
+                    if (Boolean(get(result,'0.success'))) {
```

Comment:
Interesting, I would not think using index in path like this would work.
Since you check that result[0] exists, why using get here instead of 'if (result[0].success)'? Boolean(undefined) returns false anyway.